Resources/Prototypes/Entities/Mobs/NPCs/specimen.yml

```diff
@@ -0,0 +1,18 @@
+- type: entity
+  name: Crude homunculus
+  parent: CatMob_Content
+  id: specimen_basic
+  description: I think it preffers to be called "Dmitriy"
```

Comment:
Cringe.

Reply:
I'll have to replace it with something probably, yeah